src/Compilers/CSharp/Portable/FlowAnalysis/PreciseAbstractFlowPass.cs

```diff
@@ -1293,6 +1293,11 @@ public override BoundNode VisitLiteral(BoundLiteral node)
             return null;
         }
 
+        public override BoundNode VisitMethodToken(BoundMethodToken node)
+        {
+            return null;
```

Comment:
Nope. I discovered a need to override this method by failing an assertion in a default path.

This method will disappear if we switch to the high-bit trick suggested by @tmat . I'm warming up to that.